crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -3364,21 +3326,10 @@ impl AuthorityPerEpochStore {
             non_randomness_roots.extend(roots.into_iter());
 
             if let Some(randomness_round) = randomness_round {
-                let key = TransactionKey::RandomnessRound(self.epoch(), randomness_round);
-
-                // During crash recovery, the randomness update transaction may already have been
```

Comment:
not sure I fully understand, why is this no longer necessary?

Reply:
Because it is now enqueued for execution directly by consensus handler just like any other transaction.